### DISCUSSION TITLE
[HELP] Feature not 100% working (okay nevermind it is not working at all i suppose)

### DISCUSSION THREAD

**Discussion by LohGeeTec** [Daksh777/SpotifyNoPremium#40] [Q&A]
Hello, users and dev(s). I want to report something going on my computer, about SpotifyNoPremium. Please note i am just a user, not a dev. 

I do not really know what is the version of spotify i have, all i know is my computer specs and the SpotifyNoPremium client is up to date. 
Before writing this post i only see visual ads but now i am listening to audio ads. The upgrade button still there, visual and audio ads, plus maybe limited skip. 

Any fix for this? Or maybe, if you have alternatives, can you suggest one while waiting for the fix? 

Thanks.

**Comment by Daksh777** [comment]
Hi @LohGeeTec
Can you please tell your Spotify version?
